src/Files.App.Controls/ThemedIcon/ThemedIcon.xaml

```diff
@@ -150,8 +150,8 @@
 								</VisualState>
 								<VisualState x:Name="Custom">
 									<VisualState.Setters>
-										<Setter Target="PART_OutlinePath.Fill" Value="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=Color}" />
-										<Setter Target="PART_FilledPath.Fill" Value="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=Color}" />
+										<Setter Target="PART_OutlinePath.Fill" Value="{Binding Color, Mode=OneTime}" />
```

Comment:
Shouldn't `mode` be `OneWay`?

Reply:
There are some issues with binding because of how the `Color` property is implemented. Still I just found out my current fix is causing another issue in the home page...
I'll mark this as `Ready` as soon as I find a solution